addons/core/translations/resources/en-us.yaml

```diff
@@ -925,6 +925,15 @@ worker:
       description: Create new api worker tags. You can only add API tags in the admin UI or CLI. Config tags are added in the worker config file.
       success: Tag(s) successfully created
     description: Tags are used to define the type of worker and to make it distinguishable amongst other workers.
+    remove:
+      title: Remove worker tag?
+      success: Tag successfully removed
+      warning: will be removed but removing these may affect certain targets
+      confirm:
+        label: Confirm removal
+        help: 'Enter “REMOVE” to confirm — this is case-sensitive.'
```

Comment:
just curious- why are we doing this pattern over showing a confirmation modal with just delete or cancel as that's what HDS [follows](https://helios.hashicorp.design/components/modal#critical).

Reply:
That is a great question that I do not have an answer for. I can talk with design about this.

Reply:
Talked with design and product. Since removing a worker tag can also affect a target that might be using that tag.